code/sorting/src/cocktail_sort/README.md

```diff
@@ -0,0 +1,7 @@
+
+
+Cocktail selection sort, also known as cocktail shaker sort, is a variant on bubble sort.
```

Comment:
Please format this file better like the other sorting descriptions in their folders!